server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -96,7 +96,8 @@ public RemoteFsTranslog(
         }
     }
 
-    public static void download(TranslogTransferManager translogTransferManager, Path location) throws IOException {
+    public static void download(TranslogTransferManager translogTransferManager, Path location, FileTransferTracker fileTransferTracker)
```

Comment:
Can `FileTransferTracker` be fetched from `TranslogTransferManager`?

Reply:
Yes, made the change.